lib/cylc/gui/updater_tree.py

```diff
@@ -617,70 +617,69 @@ def _update_model( self, model, columns, path, row_id, data,
         iter_ = model.append(parent_iter, data)
         return iter_
 
-    def _get_row_id( self, model, rpath ):
-        # Record a row's first two values.
-        riter = model.get_iter( rpath )
-        point_string = model.get_value( riter, 0 )
-        name = model.get_value( riter, 1 )
+    def _get_row_id(self, model, rpath):
+        """Record a row's first two values."""
+        riter = model.get_iter(rpath)
+        point_string = model.get_value(riter, 0)
+        name = model.get_value(riter, 1)
         return (point_string, name)
 
-    def _add_expanded_row( self, view, rpath, expand_me ):
-        # Add user-expanded rows to a list of rows to be expanded.
+    def _add_expanded_row(self, view, rpath, expand_me):
+        """Add user-expanded rows to a list of rows to be expanded."""
         model = view.get_model()
-        row_iter = model.get_iter( rpath )
-        row_id = self._get_row_id( model, rpath )
-        if (not self.autoexpand or
-            row_id not in self._last_autoexpand_me):
-            expand_me.append( row_id )
+        row_iter = model.get_iter(rpath)
+        row_id = self._get_row_id(model, rpath)
+        if not self.autoexpand or row_id not in self._last_autoexpand_me:
+            expand_me.append(row_id)
         return False
 
-    def _get_user_expanded_row_ids( self ):
+    def _get_user_expanded_row_ids(self):
         """Return a list of user-expanded row point_strings and names."""
         names = []
         model = self.ttreeview.get_model()
         if model is None or model.get_iter_first() is None:
             return names
-        self.ttreeview.map_expanded_rows( self._add_expanded_row, names )
+        self.ttreeview.map_expanded_rows(self._add_expanded_row, names)
         return names
 
-    def _expand_row( self, model, rpath, riter, expand_me ):
+    def _expand_row(self, model, rpath, riter, expand_me):
         """Expand a row if it matches expand_me point_strings and names."""
-        point_string_name_tuple = self._get_row_id( model, rpath )
+        point_string_name_tuple = self._get_row_id(model, rpath)
         if point_string_name_tuple in expand_me:
-            self.ttreeview.expand_to_path( rpath )
+            self.ttreeview.expand_to_path(rpath)
         return False
 
-    def _update_path_info( self, row_iter, descendant_state, descendant_name ):
+    def _update_path_info(self, row_iter, descendant_state, descendant_name):
         # Cache states and names from the subtree below this row.
```

Comment:
This is not required by the `pep8` command. (It will get picked up by `pylint`, however.) If I remember correctly, I have only done conversion of 1st comments to doc strings when:
* The line is reported to be problematic by `pep8` for whatever reason.
* The comment is a sensible doc string.